game-core/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -772,14 +767,19 @@ private void addKeyBindings(final KeyBindingSupplier... keyBindings) {
 
   @Override
   public Map<KeyStroke, Runnable> get() {
-    return Collections.singletonMap(
+    final Map<KeyStroke, Runnable> bindings = new HashMap<>();
+    bindings.put(
         KeyStroke.getKeyStroke(KeyEvent.VK_Z, InputEvent.CTRL_DOWN_MASK),
         () ->
             gameCenterPanel.setDividerLocation(
                 (gameCenterPanel.getDividerLocation()
                         <= gameCenterPanel.getMaximumDividerLocation())
                     ? 1.0
                     : gameCenterPanel.getLastDividerLocation()));
+    bindings.put(
+        KeyBindingSupplier.fromKeyEventCode(KeyEvent.VK_L),
+        () -> FlagDrawMode.toggleNextDrawMode(mapPanel));
+    return bindings;
```

Comment:
Since java 9 there exists Map.of to create maps with a small amount of keys

Reply:
Let's leave that for a future refactor please, perhaps we can go across the board and apply that update to more places than just here.